common/configuration/projectconfig_resolver.go

```diff
@@ -36,7 +35,7 @@ func (p ProjectConfigResolver[R]) Get(ctx context.Context, ref Ref) (*url.URL, e
 	}
 	key, ok := mapping[ref.Name]
 	if !ok {
-		return nil, fmt.Errorf("no such key %q: %w", ref.Name, ErrNotFound)
+		return nil, ErrNotFound
```

Comment:
This is a no-op - `errors.Is()` will search the error tree, so will be true for the previous code.

Reply:
ahh yeah I was unsure :(